threedi_modelchecker/checks/base.py

```diff
@@ -179,7 +179,7 @@ def get_invalid(self, session):
 
     def description(self):
         if len(self.columns) > 1:
-            return f"columns {[c.name for c in self.columns]} in table {self.table.name} should be unique together"
+            return f"columns {sorted(list({c.name for c in self.columns}))} in table {self.table.name} should be unique together"
```

Comment:
`sorted()` levert al een lijst op en accepteert ook sets:

```suggestion
            return f"columns {sorted({c.name for c in self.columns})} in table {self.table.name} should be unique together"
```

Reply:
Weer wat geleerd, ik zal het aanpassen.